src/main/webapp/css/style.css

```diff
@@ -23,14 +18,10 @@ h1 {
   color: #0F9D58;
 }
 
-.auth-button {
-  font: 400 15px Roboto, Arial, sans-serif;
-  color: white;
-  background: #4d90fe;
-  border: 1px #3079ED solid;
-  padding: 5px 10px;
-  border-radius: 4px;
-  text-decoration: none;
+.searchbox {
+  width: 100vh;
```

Comment:
If you just want "as large as my container lets me", do 100%. You don't actually mean 100vh - for example, on the SRP, this wouldn't leave any room for the logo, even if it were possible.